ports/esp32s2/boards/esp32s2-saola-1.cfg

```diff
@@ -0,0 +1,41 @@
+# The ESP32-S2 only supports JTAG.
```

Comment:
Where did this file come from? Does it have a license?

Reply:
I was able to find that it is a copy or near-copy of a file from openocd-esp32.  The overall openocd project is GPL licensed.  These cfg files do not have an alternate or exception notice posted.

https://github.com/espressif/openocd-esp32/blob/83c424a784f16cf47322da52f0ed42fddf7d85fb/tcl/target/esp32s2.cfg

FWIW the esp32s2.cfg file is also installed when install esp-idf, e.g., to $(HOME)/.espressif/tools/openocd-esp32/v0.10.0-esp32-20200420/openocd-esp32/share/openocd/scripts/target/esp32s2.cfg and/or $(HOME)/.local/idf_tools/tools/openocd-esp32/v0.10.0-esp32-20200420/openocd-esp32/share/openocd/scripts/target/esp32s2.cfg

Reply:
I obtained the file from the script location noted by @jelpler on my machine (the path is also obtainable by using $OPENOCD_SCRIPTS after running the export script). It is the same as the esp32s2.cfg file but with the essential difference of adding the `adapter_khz 1000` line, without which debugging will fail. I don't know why they don't include it by default.

I assumed it would have the same Apache2 licensing as the rest of the project. Do I need to take it out?

Reply:
I'd err on the side of caution and remove it. Instead document where to find it in the IDF and that the modification may be needed.

Reply:
@tannewt I opted to move the documentation to a new PR since we didn't have a Readme for the port: #3417